sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
 # Change Log azure-storage-blob
 
-## Version 12.0.0-preview.2:
+## Version 12.0.0-preview.3 (2019-09-10):
+For details on the Azure SDK for Java (September 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview3-java).
+
+- Added tracing telemetry on maximum overload API.
+- Throw `UnexpectedLengthException` when the upload body doesn't match the expected input length.
+- Added validation policy to check the equality of request client ID between request and response.
+- Updated to use service version 2019-02-02.
+- Added dependency to azure-storage-common.
```

Comment:
Is this an internal pkg? Does customer need to use this one?

Reply:
Is a package that is available to the public